### PR TITLE
Add changelog to launcher

### PR DESCRIPTION
**Pull request by struga0258**

### CODE REVIEW

**Review by timgerstel — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Zlux Server Framework Changelog
```
Comment (DivergentEuropeans) [inline CHANGELOG.md]: Change to "Zowe Launcher Changelog"

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+All notable changes to the Zlux Server Framework package will be documented in this file..
```
Comment (DivergentEuropeans) [inline CHANGELOG.md]: Change to "[...] notable changes to the Zowe Launcher will be documented"

**Review by DivergentEuropeans — DISMISSED** [review]
Missed some potential replaces from copy-paste

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Zowe Launcher Changelog
+
+All notable changes to the Zowe Launcher package will be documented in this file..
+This repo is part of the app-server Zowe Component, and the change logs here may appear on Zowe.org in that section.
+
+## 2.14.0
```
Comment (1000TurquoisePogs) [inline CHANGELOG.md]: this version does not exist.

**Review by 1000TurquoisePogs — DISMISSED** [review]

**Review by DivergentEuropeans — APPROVED** [review]

**Pull request merged**